pkgs/development/interpreters/python/cpython/2.7/default.nix

```diff
@@ -335,8 +335,9 @@ in with passthru; stdenv.mkDerivation ({
       license = lib.licenses.psfl;
       platforms = lib.platforms.all;
       maintainers = with lib.maintainers; [ fridh ];
-      # Higher priority than Python 3.x so that `/bin/python` points to `/bin/python2`
-      # in case both 2 and 3 are installed.
-      priority = -100;
+      knownVulnerabilities = [
+        "CVE-2020-26116" # CVSS 7.2
+        "CVE-2020-27619" # CVSS 9.8
+      ];
```

Comment:
Resolved by #203362, please drop this commit.

Reply:
Changed the content as per comment on https://github.com/NixOS/nixpkgs/pull/201859#issuecomment-1328520336 and other opinions/suggestions that have come up in the thread.